db/migrate/20200712150048_devise_invitable_add_to_users.rb

```diff
@@ -0,0 +1,32 @@
+class DeviseInvitableAddToUsers < ActiveRecord::Migration[6.0]
+  def up
+    safety_assured do
```

Comment:
I added this to get this to stop yelling at me after the generated migration had issues.... but this isn't the clean way to deal with this. I'll come back to this.

Reply:
Actually in looking at this I think it's fine to use here. This seems pretty safe and straightfoward I think.